src/components/message-input/mentions/index.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 
 import { UserForMention } from '../utils';
 import { bemClassName } from '../../../lib/bem';
+import { highlightFilter } from '../../messenger/lib/utils';
```

Comment:
shall move`highlightFilter` into a higher up lib folder and update import paths where used in a follow up PR towards end of sprint